packages/next-on-pages/src/buildApplication/buildApplication.ts

```diff
@@ -130,4 +129,6 @@ async function prepareAndBuildWorker(
 	);
 
 	cliSuccess(`Generated '${outputtedWorkerPath}'.`);
+
+	await buildMetadataFiles(outputDir, { staticAssets });
```

Comment:
Is there a specific reason why you added this here? 
could you move this right below line 88? (`const staticAssets = await getVercelStaticAssets();`)

(or in any case I don't think this should be below the successful generation message)

Reply:
> Is there a specific reason why you added this here?

consistenty with how it worked before i moved it into this function. it was previously after everything

Reply:
i moved it, but not quite to the place you suggested

Reply:
ok that works for me, thanks 😄